### PR TITLE
added filter and pagination for Table in AdminModule

### PR DESCRIPTION
**Pull request by ShpakKate**

### CODE REVIEW

File: src/app/admin/admin.module.ts [hunk 1]
```diff
@@ -11,9 +11,16 @@ import { EditingUserDataComponent } from './components/editing-user-data/editing
 import { DeletingUserComponent } from './components/deleting-user/deleting-user.component';
 import { ReactiveFormsModule } from '@angular/forms';
 import { ListOfUsersComponent } from './components/list-of-users/list-of-users.component';
+import {MatSortModule} from "@angular/material/sort";
```
Comment (burymm) [inline src/app/admin/admin.module.ts]: spaces missed in brackets

File: src/app/admin/components/list-of-users/list-of-users.component.ts [hunk 2]
```diff
@@ -14,38 +14,44 @@ import { MatPaginator } from '@angular/material/paginator';
...
+    private auth: AuthenticationService,
+    public dialog: MatDialog
+  ) {}
 
   ngOnInit() {
     this.loadData().subscribe();
   }
 
+  ngAfterViewInit() {
+    setTimeout(() => this.dataSource.paginator = this.paginator);
```
Comment (burymm) [inline src/app/admin/components/list-of-users/list-of-users.component.ts]: Why do u need setTimeout?

File: src/app/components/news/news.component.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ import { Router } from '@angular/router';
 })
 export class NewsComponent implements OnInit {
   newsList$: Observable<News[]> = of([]);
+  array: any = 'text'
```
Comment (burymm) [inline src/app/components/news/news.component.ts]: bad variable name

File: src/app/components/newsitem/newsitem.component.ts [hunk 4]
```diff
@@ -10,6 +10,7 @@ import { News } from '../../../shared/model/news.model';
 })
 export class NewsitemComponent implements OnInit {
   @Input() item!: News;
+  @Input() arr!: any;
```
Comment (burymm) [inline src/app/components/newsitem/newsitem.component.ts]: not clear variable name

File: src/app/admin/admin.module.ts (hunk 1 above)
Comment (ShpakKate) [reply src/app/admin/admin.module.ts]: +

File: src/app/admin/components/list-of-users/list-of-users.component.ts (hunk 2 above)
Comment (ShpakKate) [reply src/app/admin/components/list-of-users/list-of-users.component.ts]: +

File: src/app/components/news/news.component.ts (hunk 3 above)
Comment (ShpakKate) [reply src/app/components/news/news.component.ts]: +

File: src/app/components/newsitem/newsitem.component.ts (hunk 4 above)
Comment (ShpakKate) [reply src/app/components/newsitem/newsitem.component.ts]: +

**Review by Dewlar — APPROVED** [review]

**Pull request merged**